modules/claims_api/app/swagger/claims_api/forms/form_526_response_swagger.rb

```diff
@@ -229,7 +229,185 @@ class Form526ResponseSwagger
                   items do
                     key :type, :string
                     key :example, 'Hardship'
-                    key :enum, ['Hardship', 'Seriously Injured/Very Seriously Injured', 'Terminally Ill', 'Homeless', 'Purple Heart', 'POW', 'Medal of Honor', 'Amyotrophic Lateral Sclerosis', 'Emergency Care']
+                    key :enum, %w[
```

Comment:
`%w` won't work here i believe, since there are spaces within the values.
tested my theory out here and it does seem to be the case
```
2.6.6 :001 > (%w[1151 P&T 38 USC 1151]).count
 => 5 
```
These will need to be converted back to the original array of strings format.